### ISSUE TITLE
Add support for filtering again.

### ISSUE THREAD

**Issue by onodera-punpun** [arybczak/ncmpcpp#112]
This was the ncmpcpp feature I used the most, and really, in my opinion search forward/backward doesn't even come close in terms of usability. Before I could easily filter my playlist for a specific artist, song, or whatever. Now I first need to think if the song is "backwards" or "forwards", search for it, go though the search results, etc. etc.

**Comment by jcherqui** [comment]
+1

**Comment by jcherqui** [comment]
+1

**Comment by onodera-punpun** [comment]
I don't think the dev uses git tbh..

**Comment by privatemajory** [comment]
+1

**Comment by jcherqui** [comment]
:+1:

**Comment by arybczak** [comment]
Ironically at some point I realized that indeed filtering is not as convenient for quick search as forward/backward searching. Combine that with mild frustration I had when I was on the receiving end of people removing functionality from their apps for no good reason and I decided to bring this feature back. I'm sorry for making this mistake and that it took so long to correct.

Reimplemented in 0.8_dev.
